### PR TITLE
testing: enable configuration of log level and time stamps

### PR DESCRIPTION
**Pull request by pohly**
Both may or may not be useful. Now the user can decide.

### CODE REVIEW

File: testing/test.go [hunk 1]
```diff
@@ -33,6 +33,31 @@ func NewTestLogger(t *testing.T) logr.Logger {
 	return logr.New(l)
 }
 
+// Options carries parameters which influence the way logs are generated.
+type Options struct {
+	// LogTimestamps tells the logger to add a "ts" key to log
```
Comment (thockin) [inline testing/test.go:38]: Comment and code don't match (names are different)

Comment (thockin) [reply testing/test.go:38]: also broken in funcr, if you don't mind fixing both

Comment (pohly) [reply testing/test.go]: I wonder whether we should have a check for this. I completely missed it.

Anyway, fixed in both places.

Comment (thockin) [reply testing/test.go]: I am surprised no linter catches it.

Comment (pohly) [reply testing/test.go]: Answer in issue #82...

**Pull request merged**